contracts/DomainRegistry.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "hardhat/console.sol";
+import "./Ownable.sol";
+
+contract DomainRegistry is Ownable {
+    uint256 public totalDomains;
```

Comment:
What's the point for this metric? How can the "dirty" number of domains (with released as well) be useful?
-1 point